src/slicing/solver.rs

```diff
@@ -602,14 +618,15 @@ pub fn slice_unsat_search<'ctx>(
         limits_ref.check_limits()?;
 
         match check_proof_seed(&all_variables, prover, limits_ref, &seed) {
-            ProveResult::Proof => {
+            Ok(ProveResult::Proof) => {
                 // now start the shrinking, then block up
-                let res = exploration.shrink_block_unsat(seed, |seed| {
-                    match check_proof_seed(&all_variables, prover, limits_ref, seed) {
-                        ProveResult::Proof => Some(unsat_core_to_seed(prover, &all_variables)),
-                        ProveResult::Counterexample | ProveResult::Unknown(_) => None,
-                    }
-                });
+                let res_seed = match check_proof_seed(&all_variables, prover, limits_ref, &seed) {
```

Comment:
Wait, I don't think this is the same behavior! `shrink_block_unsat` should call `check_proof_seed` as before!